src/pages/DetailsPage.js

```diff
@@ -101,7 +101,15 @@ const DetailsPage = ({personalDetails, route}) => {
                                     <Text style={[styles.textP]} numberOfLines={1}>
                                         {moment().tz(details.timezone.selected).format('LT')}
                                         {' '}
-                                        {moment().tz(details.timezone.selected).zoneAbbr()}
+                                        {isNaN(moment().tz(details.timezone.selected).zoneAbbr()) ? ( 
```

Comment:
Since we are using `moment().tz(details.timezone.selected)` a lot, let's just store it in a variable we can reuse.

Reply:
@Gonals I did not get it. Are you saying to store the entire variable in another variable?

Reply:
Yes. Something like `const timezone = moment().tz(details.timezone.selected);`, and then you can just reuse `timezone`

Reply:
Now changed. Please check.